qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -50,8 +50,8 @@ def run_cephfs_shell_cmd(self, cmd, mount_x=None, shell_conf_path=None,
         args.extend(("--", cmd))
 
         log.info("Running command: {}".format(" ".join(args)))
-        return mount_x.client_remote.run(args=args, stdout=BytesIO(),
-                                         stderr=BytesIO(), stdin=stdin)
+        return mount_x.client_remote.run(args=args, stdout=StringIO(),
+                                         stderr=StringIO(), stdin=stdin)
```

Comment:
Here's what I suggest as a compromise: make it caller defined (with a reasonable default). So instead:

````
def run_cephfs_shell_cmd(self, cmd, mount_x=None, shell_conf_path=None, opts=None, stdin=None, stdout=StringIO(), stderr = StringIO()):
    ...
    return mount_x.client_remote.run(args=args, stdout=stdout, stderr=stderr, stdin=stdin)
````

Ditto for `run_cephfs_shell_script`.

Reply:
Better yet, IMO. Made the change.